src/Thentos/Smtp.hs

```diff
@@ -6,6 +6,7 @@ module Thentos.Smtp
     ( sendUserConfirmationMail
     , sendUserExistsMail
     , sendPasswordResetMail
+    , sendEmailChangeConfirmMail
```

Comment:
you're right.  please don't change it.  (i don't have an opinion on 'confirmation' vs. 'confirm'.)